### ISSUE TITLE
Javafx validation failed after updating recaf.

### ISSUE THREAD

**Issue by Liticane** [Col-E/Recaf-Launcher#13]
```
Attempting to resolve installed Recaf version...
Found Recaf's build config, checking contents...
 - MAVEN_GROUP = software.coley
 - MAVEN_NAME = recaf-core
 - VERSION = 4.0.0-SNAPSHOT
 - GIT_REVISION = 1
 - GIT_SHA = ebfd9b9bb3e856eb06985493e65efe92de7b2c94
 - GIT_DATE = 2024 04/15 19:52
 - GIT_BRANCH = dev4
 - BUILD_DATE = 2024 04/15 19:53
 - BUILD_UNIX_TIME = 1713210800811
 - DIRTY = 0
Attempting to resolve JFX version...
No JavaFX version class found in the current classpath
Java Properties:
 - java.home = D:\Program Files\Zulu\zulu-22
 - java.version = 22.0.1
 - java.vm.vendor = Azul Systems, Inc.
 - java.vm.version = 22.0.1+8
 - javafx.platform = win
 - javafx.version = -1
 - os.arch = amd64
 - os.bitness = 64
 - os.name = Windows 11
 - os.processors = 8
 - os.version = 10.0
 - recaf.version = 4.0.0-SNAPSHOT

Looking in 'C:\Users\Liticane\AppData\Roaming\Recaf' for Recaf/dependencies...
Running Recaf '4.0.0-SNAPSHOT' with JavaFX '23-ea+3:win'
23:52:30.171 [JFXValidation/main] ERROR: JFX validation failed, could not find 'VersionInfo' class
java.lang.ClassNotFoundException: com.sun.javafx.runtime.VersionInfo
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:525)
        at java.base/java.lang.Class.forName0(Native Method)
        at java.base/java.lang.Class.forName(Class.java:413)
        at java.base/java.lang.Class.forName(Class.java:404)
        at software.coley.recaf.util.JFXValidation.validateJFX(JFXValidation.java:33)
        at software.coley.recaf.Main.main(Main.java:70)
```

**Comment by xcfrg** [comment]
likely dupe of #12

**Comment by Col-E** [comment]
Update the launcher, like mentioned above probably got resolved in `0.4.1`

I'll re-open if this isn't resolved by updating.
